tools/gen_selectedops.py

```diff
@@ -1,80 +1,99 @@
-import os
-import sys
+#!/usr/bin/env python3
+
+import argparse
+import pathlib
 
 OPMAP_TO_CMAKE_FLAGS = {
-    'BlingFireSentenceBreaker': 'OCOS_ENABLE_BLINGFIRE',
-    'GPT2Tokenizer': 'OCOS_ENABLE_GPT2_TOKENIZER',
-    'WordpieceTokenizer': 'OCOS_ENABLE_WORDPIECE_TOKENIZER',
-    'StringConcat': 'OCOS_ENABLE_TF_STRING',
-    'StringECMARegexReplace': 'OCOS_ENABLE_TF_STRING',
-    'StringECMARegexSplitWithOffsets': 'OCOS_ENABLE_TF_STRING',
-    'StringEqual': 'OCOS_ENABLE_TF_STRING',
-    'StringToHashBucket': 'OCOS_ENABLE_TF_STRING',
-    'StringToHashBucketFast': 'OCOS_ENABLE_TF_STRING',
-    'StringJoin': 'OCOS_ENABLE_TF_STRING',
-    'StringLength': 'OCOS_ENABLE_TF_STRING',
-    'StringLower': 'OCOS_ENABLE_TF_STRING',
-    'StringRegexReplace': 'OCOS_ENABLE_RE2_REGEX',
-    'StringRegexSplitWithOffsets': 'OCOS_ENABLE_RE2_REGEX',
-    'StringSplit': 'OCOS_ENABLE_TF_STRING',
-    'StringToVector': 'OCOS_ENABLE_TF_STRING',
-    'StringUpper': 'OCOS_ENABLE_TF_STRING',
-    'SegmentExtraction': 'OCOS_ENABLE_MATH',
-    'StringMapping': 'OCOS_ENABLE_TF_STRING',
-    'VectorToString': 'OCOS_ENABLE_TF_STRING',
-    'MaskedFill': 'OCOS_ENABLE_TF_STRING',
-    'BertTokenizer': 'OCOS_ENABLE_BERT_TOKENIZER',
-    'BasicTokenizer': 'OCOS_ENABLE_BERT_TOKENIZER',
-    'BertTokenizerDecoder': 'OCOS_ENABLE_BERT_TOKENIZER',
-    'SentencepieceTokenizer': 'OCOS_ENABLE_SPM_TOKENIZER',
-    'ImageDecode': 'OCOS_ENABLE_VISION',
-    'ImageEncode': 'OCOS_ENABLE_VISION',
+    "BlingFireSentenceBreaker": "OCOS_ENABLE_BLINGFIRE",
+    "GPT2Tokenizer": "OCOS_ENABLE_GPT2_TOKENIZER",
+    "WordpieceTokenizer": "OCOS_ENABLE_WORDPIECE_TOKENIZER",
+    "StringConcat": "OCOS_ENABLE_TF_STRING",
+    "StringECMARegexReplace": "OCOS_ENABLE_TF_STRING",
+    "StringECMARegexSplitWithOffsets": "OCOS_ENABLE_TF_STRING",
+    "StringEqual": "OCOS_ENABLE_TF_STRING",
+    "StringToHashBucket": "OCOS_ENABLE_TF_STRING",
+    "StringToHashBucketFast": "OCOS_ENABLE_TF_STRING",
+    "StringJoin": "OCOS_ENABLE_TF_STRING",
+    "StringLength": "OCOS_ENABLE_TF_STRING",
+    "StringLower": "OCOS_ENABLE_TF_STRING",
+    "StringRegexReplace": "OCOS_ENABLE_RE2_REGEX",
+    "StringRegexSplitWithOffsets": "OCOS_ENABLE_RE2_REGEX",
+    "StringSplit": "OCOS_ENABLE_TF_STRING",
+    "StringToVector": "OCOS_ENABLE_TF_STRING",
+    "StringUpper": "OCOS_ENABLE_TF_STRING",
+    "SegmentExtraction": "OCOS_ENABLE_MATH",
+    "StringMapping": "OCOS_ENABLE_TF_STRING",
+    "VectorToString": "OCOS_ENABLE_TF_STRING",
+    "MaskedFill": "OCOS_ENABLE_TF_STRING",
+    "BertTokenizer": "OCOS_ENABLE_BERT_TOKENIZER",
+    "BasicTokenizer": "OCOS_ENABLE_BERT_TOKENIZER",
+    "BertTokenizerDecoder": "OCOS_ENABLE_BERT_TOKENIZER",
+    "SentencepieceTokenizer": "OCOS_ENABLE_SPM_TOKENIZER",
+    "ImageDecode": "OCOS_ENABLE_VISION",
+    "ImageEncode": "OCOS_ENABLE_VISION",
```

Comment:
Sorry - I created the mismatch with names at some point. Should be DecodeImage and EncodeImage to match the actual operator names. I can fix in a separate PR if you like.

Reply:
got it. np, will fix it here